_data/simple-icons.json

```
@@ -8219,6 +8219,12 @@
             "hex": "39457E",
             "source": "https://github.com/tpf/marketing-materials/blob/6765c6fd71bc5b123d6c1a77b86e08cdd6376078/images/onion-logo/tpf-logo-onion.svg"
         },
+        {
+            "title": "Persistent",
+            "hex": "FD5F07",
+            "source": "https://www.persistent.com/wp-content/uploads/2021/03/Persistent-Logo-Kit.zip",
```

According to the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), the link to the branding page and not the link to the ZIP archive should be used:

> If the SVG is sourced from:
>
> - **Branding page**: For an SVG from a branding page the source URL should link to the branding page and not the image, PDF, or archive (such as `.zip`) file.

```suggestion
            "source": "https://www.persistent.com/company-overview/branding-guidelines/#nav-persistent-logo",
```

---

This topic hasn't been resolved.